CONTRIBUTING.adoc

```diff
@@ -0,0 +1,108 @@
+= How to contribute to Rudder
+
+Rudder is open to all kinds of contribution:
+
+* Bug reports and fixes
+* Documentation
+* Features ideas
+* New features
+
+If you want to contribute, the best way to start is to come and chat with us
+on https://chat.rudder.io[chat.rudder.io]. You can also communicate by email
+using the  https://www.rudder-project.org/mailman/listinfo/rudder-dev[rudder-dev]
+mailing list.
+
+NOTE: Contribution to Rudder requires a https://www.rudder.io/en/expand/contribute/#panel-2422-8-0-1[CLA/CCLA].
+
+== Development workflow
+
+We maintain several Rudder major versions in separate branches.
+
+* `branches/rudder/5.0` is Rudder 5.0, from which 5.0 minor release will be tagged
+* `master` is the next Rudder version, not branched yet
+
+NOTE: Some tooling repositories only have a `master` branch.
+
+All changes need to be done in the oldest relevant branch (i.e.
```

Comment:
s/changes need to/bug correction should/
And perhaps something explaining that we can assist them if they fear to deal with that oldest-branch rule.